src/Components/ClusterRules/ClusterRules.spec.ct.js

```diff
@@ -26,54 +67,393 @@ describe('cluster rules table', () => {
       req.destroy();
     });
 
-    cy.fixture(
-      'api/insights-results-aggregator/v1/clusters/41c30565-b4c9-49f2-a4ce-3277ad22b258/report.json'
-    ).then((reports) => {
-      mount(
-        <IntlProvider locale="en">
-          <Provider store={getStore()}>
-            <MemoryRouter
-              initialEntries={[
-                '/clusters/41c30565-b4c9-49f2-a4ce-3277ad22b258',
-              ]}
-              initialIndex={0}
-            >
-              <Route path="/clusters/:clusterId">
-                <ClusterRules reports={reports} />
-              </Route>
-            </MemoryRouter>
-          </Provider>
-        </IntlProvider>
-      );
-    });
+    mount(
```

Comment:
If we use the same components in the `mount`, we can extract it from this `beforeEach` block to the separate `before` block. This way, you wouldn't need to copy/paste the same component for each `describe` block. The same works for interceptors (and actually everything which all `describe` scenarios have in common.

Reply:
We are using the same component, but `beforeEach`  gives us some nice features, as resting filters and sorting without requiring a cleanup after the test